### ISSUE TITLE
Ability to create sub licenses.

### ISSUE THREAD

**Issue by lucaskent** [soflyy/oxygen-bugs-and-features#1699]
Can we have the ability to create and assign a license to a particular site? This way if we build a website for any client of ours, they may not in turn reuse the license on their separate other projects.

**Comment by lucaskent** [comment]
Also are you considering uploading the plugin on wordpress plugins directory?

**Comment by kalaio** [comment]
+1 for this feature

**Comment by shmaltz** [comment]
I like this idea - and it should be tied to the domain.

**Comment by cstrouse** [comment]
You can already authorize the license per-domain and deactivate unauthorized domains.

<img width="826" alt="Screen Shot 2020-11-08 at 2 47 00 PM" src="https://user-images.githubusercontent.com/849044/98485146-6c130780-21d1-11eb-9a1b-8c9cf1f7aca0.png">

**Comment by lucaskent** [comment]
> You can already authorize the license per-domain and deactivate unauthorized domains.
> 
> <img alt="Screen Shot 2020-11-08 at 2 47 00 PM" width="826" src="https://user-images.githubusercontent.com/849044/98485146-6c130780-21d1-11eb-9a1b-8c9cf1f7aca0.png">

yes but if somebody figures out the key from their website, they can use the key on other sites as well without my knowledge so until i go to account and check, i won't find anything. At leas some kind of notification when somebody activates a new site, would be helpful.

**Comment by lucaskent** [comment]
Also one more request/suggestion, i think the widget area should be enabled by default as opposed to using a plugin to enable and create widget areas, i think this should be a native in built functionality as users are so used to wordpress widgeting system.

**Comment by lucaskent** [comment]
By the way i didn't find any option to wrap text around image option, can you pls advise?
